styles/app.css

```diff
@@ -0,0 +1,139 @@
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.navbar {
+  background: #2CC5E1;
+  overflow: auto;
+  margin: auto;
+}
+
+.navbar-left {
+ float: left;
+ display: inline-block;
+ color: white;
+padding-bottom: 1rem;
```

Comment:
You have padding in a few places that may make it hard to come back later and find which element is applying whitespace.